api/controllers/userController.js

```diff
@@ -20,6 +20,18 @@ const signUp = async (req, res) => {
     }
 }
 
+const signIn = async (req, res) => {
+    const {email, password} = req.body
+
+    try{
+        const accessToken = await userService.signIn(email, password)
+        res.status(200).json({ accessToken })
+    }catch (err){
+        res.status(err.statusCode|| 400).json({message: err.message});
+    }
+}
```

Comment:
request의 body의 데이터도 try 구문 안에서 실행할 수 있게 수정해주세요!
res.json의 인수로 object 타입의 반환값을 넣어 json 형태로 파싱할 수 있게 해주면 좋을 것 같습니다.

`console.log(err)`을 통해 어떤 에러가 반환되는지 로그를 확인할 수 있습니다.

```suggestion
    try{
        const {email, password} = req.body
        const accessToken = await userService.signIn(email, password)
        
        return res.status(200).json({ accessToken: accessToken })
    } catch (err) {
        console.log(err)
        res.status(err.statusCode|| 400).json({ message: err.message });
    }
```